docs/shipping/App360/App360.md

```diff
@@ -174,11 +174,15 @@ service:
     traces:
       receivers: [otlp]
       processors: [tail_sampling, batch]
-      exporters: [logzio/traces, spanmetrics]
+      exporters: [logzio/traces]
+    traces/spm:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [spanmetrics]
     metrics/spanmetrics:
       receivers: [spanmetrics]
       processors: [metricstransform/metrics-rename, metricstransform/labels-rename]
-      exporters: [prometheusremotewrite/spm]      
+      exporters: [prometheusremotewrite/spm]
   telemetry: 
     logs:
       level: "debug"
```

Comment:
This is not related to current changes, but can we change this to be `info` by default? `tail_sampling` processor is generating tons of debug logs that can spam the users